dist/index.html

```diff
@@ -0,0 +1 @@
+<script type="text/javascript" src="index_bundle.js"></script>
```

Comment:
@Philipotieno, the dist folder should be added to gitignore. It should not be pushed.

Reply:
I am fixing that now @IssaIan